rq/job.py

```diff
@@ -351,6 +351,7 @@ def __init__(self, id=None, connection=None, serializer=None):
         # retry_intervals is a list of int e.g [60, 120, 240]
         self.retry_intervals = None
         self.redis_server_version = None
+        self.heartbeat = None
```

Comment:
This should be renamed to `last_heartbeat`

Reply:
`job.last_heartbeat` should also be added to the docs here: https://github.com/rq/rq/blob/master/docs/docs/jobs.md